Sources/IO/XML/XMLReader/index.js

```diff
@@ -18,6 +19,13 @@ function stringToXML(xmlStr) {
   return (new DOMParser()).parseFromString(xmlStr, 'application/xml');
 }
 
+function parseArrayBuffer(buffer) {
```

Comment:
`extractAppendedData` just to prevent naming confusion with the `parseArrayBuffer`.